processor/metricstransformprocessor/README.md

```diff
@@ -0,0 +1,116 @@
+# Metrics Transform Processor <span style="color:red">**(UNDER DEVELOPMENT - NOT READY FOR USE)**</span>
+- <span style="color:red">This ONLY supports renames/aggregations **within individual metric batches**.</span> It does not do any aggregation across batches, so it is not suitable for aggregating metrics from multiple sources (e.g. multiple nodes or clients). At this point, it is only for aggregating metrics from a single source that groups its metrics for a particular time period into a single batch (e.g. host metrics from the VM the collector is running on).
```

Comment:
Is batch the the right word here? It sounds weird to me. I'm not actually sure on the right terminology, but I think a metric is defined as the aggregation of individual measurements? (in which case you could just say "This ONLY supports renames/aggregations **within individual metrics**)

(I could be wrong on that)

Reply:
Actually, this makes sense to me because at this point, all operations are done based on a metric name match, which means all within one metric